src/tests/Details.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
- [ ] Please add more integration tests that test for user interactions across the app, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Create%20integration%20tests%20for%20your%20application%20using%20the%20React%20Testing%20Library.) requirement.